ckanext/spectrum/tests/test_action_dataset_duplicate.py

```diff
@@ -19,6 +21,7 @@ def dataset():
     return call_action('package_show', id=dataset['id'])
 
 
+@pytest.mark.ckan_config('ckan.plugins', "spectrum scheming_datasets")
```

Comment:
All tests which have '@pytest.mark.ckan_config...' added use 'package_create'  somewhere within (e.g. factories.Dataset uses this action :) ) and since we add 'spectrum' plugin in our test code we validate dataset's 'type' in each of those places. Checking of 'type' requires action from 'scheming_datasets" plugin, hence it is required as well.

Reply:
If the majority of tests require scheming_datasets now it would make sense to add the plugins to the test.ini config (you can always overwrite it and remove the plugin if the minority doesn't require it).
That's just FYI as I'm not sure which count of test is greater.